components/hackathon/common/Sponsors.tsx

```diff
@@ -0,0 +1,39 @@
+import { Flex, Image, SimpleGrid } from "@chakra-ui/react";
+import { Text, TrackedLink } from "tw-components";
+
+interface ISponsorProps {
+  sponsors: {
+    name: string;
+    logo: string;
+    link: string;
+  }[];
+}
```

Comment:
same comment about the naming of the interface and extracting or array syntax for the items